src/api/RestAPI.ts

```diff
@@ -298,6 +300,7 @@ export class RestAPI extends EventEmitter {
 
         // Deprecated
         const { authToken = this.authToken, appId = this.appId, secret = this.secret, jwt = this.jwtRaw } = {
+            ...auth,
             ...(!isFormData ? data : {}),
         };
```

Comment:
We may want to remove the `Deprecated` as it is not about form data only any longer.

Also I would recommend to have `...auth` on the second line as it is an override. As formData is not initially used for this purpose it make sense for the `auth` parameter to have the last word in my humble opinion 🙂

Reply:
Yup, right! Done!